lib/generators/manageiq/plugin/templates/lib/%plugin_path%/engine.rb

```diff
@@ -1,5 +1,15 @@
 <% exploded_class_name do %>
   class Engine < ::Rails::Engine
     isolate_namespace <%= class_name %>
+
+    config.autoload_paths << root.join('lib').to_s
+
+    def vmdb_plugin?
+      true
+    end
+
+    def self.plugin_name
```

Comment:
@jrafanie Note that this plugin_name is completely different from the one defined as a generator helper methods.  This is the actual method exposed from the plugin itself.